Modules/HR/Http/Controllers/Recruitment/ApplicationController.php

```
@@ -136,28 +137,39 @@ public function index()
                 ->count();
         }
 
-        $attr['jobs'] = Job::all();
-        $attr['universities'] = University::all();
-        $attr['tags'] = Tag::orderBy('name')->get();
+        $attr['jobs'] = Cache::remember('jobs', now()->addMinutes(60), function () {
```

@Ayush8923 This will restrict the system updates for one hour. So let's say you add a tag now and it will show up after 1 hour which does not make sense to me. 

Also this implementation is good but I feel that there are lot of scope to fix the existing code and then apply caching for these things. If we apply caching right now it will just suppress the error and time. 

So for now remove this implementation and try to fix all the queries we are making to DB and just fix them. 








This seems like a fine approach but we will use it as a last thing.

---

Make sense 👍